gnosis/verification.k

```diff
@@ -99,31 +116,69 @@ module VERIFICATION
     rule 0 <=Int #bufElm(_, _)  => true
     rule #bufElm(_, _) <Int 256 => true
 
+    rule 0 <=Int X xorInt 115792089237316195423570985008687907853269984665640564039457584007913129639935 => true      requires 0<=Int X andBool X <Int pow256
+
+    rule X xorInt 115792089237316195423570985008687907853269984665640564039457584007913129639935 <Int pow256 => true  requires 0<=Int X andBool X <Int pow256 
```

Comment:
please put a comment for the big magic number.

Reply:
done